scripts/lockfile/lockfile.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+// This .go file is used to test the lockfile package (m3/src/x/lockfile)
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strconv"
+	"time"
+
+	"github.com/m3db/m3/src/x/lockfile"
+)
+
+func exitWithUsage() {
+	fmt.Printf("Usage: %s PATH SLEEP_SECONDS SHOULD_REMOVE_LOCK\n", path.Base(os.Args[0]))
```

Comment:
nit: "Usage: %s <LOCK_FILE_PATH> <NUM_SECONDS_TO_SLEEP> <SHOULD_CLEANUP_LOCK>\n"

Reply:
and then maybe one more line with a concrete example